operation/fhir-operation-validate/src/test/java/com/ibm/fhir/operation/validate/ValidateOperationTest.java

```diff
@@ -268,4 +284,218 @@ public void testProfileParmProfileUnsupportedError() {
+                    .code(IssueType.INFORMATIONAL)
+                    .details(CodeableConcept.builder()
+                        .text(string("All OK"))
+                        .build())
+                    .build();
+        Parameters output = validateOperation.doInvoke(operationContext, Patient.class, "1", null, input, resourceHelper, null);
+        OperationOutcome operationOutcome = output.getParameter().get(0).getResource().as(OperationOutcome.class);
+        assertEquals(operationOutcome.getIssue().size(), 1);
+        assertEquals(operationOutcome.getIssue().get(0), expectedOutput);
+    }
+    
+    /**
+     * Test resource-instance level validate operation when the resource with the input logicalID is not available in the database.
+     * @throws Exception 
+     */
+    @Test(expectedExceptions = { FHIROperationException.class } , expectedExceptionsMessageRegExp  = ".*Patient with id '1' was not found*")
+    public void testResourceInstanceLevelValidateForNullResource() throws Exception {
+        Patient patient = (Patient) getTestResource("Patient.json");
+        FHIRResourceHelpers resourceHelper = mock(FHIRResourceHelpers.class);
+        SingleResourceResult result = mock(SingleResourceResult.class);
+        
+        // mock and return null when resourceHelper.doRead() is invoked from validateOperation.doInvoke
+        when(result.isSuccess()).thenReturn(false);
+        when(result.getResource()).thenReturn(null);
+        when(resourceHelper.doRead(eq("Patient"), anyString())).thenAnswer(x -> result);
+        
+        FHIROperationContext operationContext =
+                FHIROperationContext.createInstanceOperationContext("validate");
+        Parameters input = Parameters.builder()
+                .build();
+        
+        Issue expectedOutput = Issue.builder()
+                    .severity(IssueSeverity.INFORMATION)
+                    .code(IssueType.INFORMATIONAL)
+                    .details(CodeableConcept.builder()
+                        .text(string("All OK"))
+                        .build())
+                    .build();
+
+
+        Parameters output = validateOperation.doInvoke(operationContext, Patient.class, "1", null, input, resourceHelper, null);
+        OperationOutcome operationOutcome = output.getParameter().get(0).getResource().as(OperationOutcome.class);
+        assertEquals(operationOutcome.getIssue().size(), 1);
+        assertEquals(operationOutcome.getIssue().get(0), expectedOutput);
+    }
+    
+    /**
+     * Read a mock resource from the input JSON file.
+     *
+     * @param path the path to JSON file
+     * @return Resource - A FHIR Resource object representation
+     * @throws Exception
+     */
+    private static Resource getTestResource(String path) throws Exception {
+        try (InputStream is = ClassLoader.getSystemResourceAsStream(path)) {
+            return FHIRParser.parser(Format.JSON).parse(is);
+        }
```

Comment:
per my other comment, I think this could just be
```
return Patient.builder().id("test").build();
```

Reply:
or, if you take a dependency on fhir-model's test-jar (which unfortunately also means depending on org.skyscreamer:jsonassert at the moment) you could be
```
return TestUtil.getMinimalResource(Patience.class);
```


here is the dependency snippet for the pom.xml in case you chose to go this route:
```xml
        <dependency>
            <groupId>${project.groupId}</groupId>
            <artifactId>fhir-model</artifactId>
            <version>${project.version}</version>
            <type>test-jar</type>
        </dependency>
        <dependency>
            <groupId>org.skyscreamer</groupId>
            <artifactId>jsonassert</artifactId>
            <scope>test</scope>
        </dependency>
```

Reply:
This has been fixed. Removed Patient.json and built up Patient resource using Patient.builder() with minimal fields as suggested in the previous review comment.